packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -765,30 +765,30 @@ export class DataStores implements IDisposable {
 			const pathParts = route.split("/");
 			const dataStoreId = pathParts[1];
 
-			// TODO: GC:Validation - Skip any routes already deleted
```

Comment:
This was actually covered by the assert below I guess?  And now we are backing down to check production first, since it's not that bad, just confusing?

Or, is this about checking against specifically already-deleted (not simply missing) nodes? Thoughts on that?

Reply:
With GC op, its possible to have mutliple GC ops with the same deleted nodes. So, there shouldn't be any asserts.
If the node is simply missing, it means GC knows about some data store that doesn't exist. That would indicate some problem but this is not the right place to throw an error or anything.

In both the case, logging seems okay. Maybe I can include whether the node was deleted or not. That would be worthwhile.

Reply:
Added `alreadyDeleted` property to the log.